src/azure-cli/azure/cli/command_modules/hdinsight/_params.py

```diff
@@ -184,7 +184,7 @@ def load_arguments(self, _):
                    help='The client AAD security group name for Kafka Rest Proxy')
 
         # Managed Service Identity
-        c.argument('assign_identity', arg_group='Managed Service Identity', validator=validate_msi,
+        c.argument('assign_identity', nargs='*', arg_group='Managed Service Identity', validator=validate_msi,
```

Comment:
add note that when the assign_identity_type is systemassigned skip this

Reply:
Done